general-configuration-guide/basic-setup.md

```diff
@@ -249,48 +225,61 @@ The following example is a walkthrough of a basic {{site.productname}} configura
 </html>
 ```
 
-#### Breakdown of the above example
+#### Break-down of the basic configuration example
 
-Select only the `textarea` with the id `myTextarea`.
-
-```js
-selector: '#myTextarea',
-```
+The following section is a break-down of the options used in the basic configuration example.
 
-Choose a Skin such as the dark skin which is included with {{site.productname}}.
+Selects the `textarea` with the id `myTextarea` to be replaced by the editor.
 
 ```js
-skin: 'dark',
+selector: '#myTextarea',
 ```
 
-Set the width and height of the editable area in pixels as numeric values.
+Sets the width and height of the editable area in pixels as numeric values.
 
 ```js
 width: 600,
 height: 300,
 ```
 
-Select the plugins that should be included on load.
+Selects the plugins to be included on load.
 
 ```js
 plugins: [
   'advlist autolink link image lists charmap print preview hr anchor pagebreak spellchecker',
   'searchreplace wordcount visualblocks visualchars code fullscreen insertdatetime media nonbreaking',
-  'save table directionality emoticons template paste'
+  'table emoticons template paste help'
 ],
 ```
 
-Set the styling of the editable area using `content_css`. The styling should be a simplified version of the website CSS. Use styles for headers (H1-H6), table layouts, margins, paddings around elements (images, paragraphs), etc.
+Selects the toolbar buttons exposed to the user. Use a comma or space as a separator.
```

Comment:
"displayed" instead of "exposed"? Exposed sounds like an API thing

Reply:
true. updated